src/app/legacy/components/Promo/heading.jsx

```diff
@@ -5,25 +5,12 @@ import { GEL_SPACING } from '#psammead/gel-foundations/src/spacings';
 import { getBodyCopy } from '#psammead/gel-foundations/src/typography';
 import { C_EBON } from '#psammead/psammead-styles/src/colours';
 
-import {
-  GEL_GROUP_3_SCREEN_WIDTH_MIN,
-  GEL_GROUP_4_SCREEN_WIDTH_MIN,
-} from '#psammead/gel-foundations/src/breakpoints';
-
 const Heading = styled.h2`
```

Comment:
May be worth using the Theme values here for the font and colour if possible. Would require passing in the `index` to determine the font, but brings it more inline with the component standards, e.g:
```suggestion
const Heading = styled.h2`
  ${props => props.theme.fontVariants.serifMedium}
  ${props =>
    props.index === 0
      ? props.theme.fontSizes.paragon
      : props.theme.fontSizes.bodyCopy}
  color: ${props => props.theme.palette.EBON};
  margin-top: 0;
  margin-bottom: 0.5rem;
`;
```

Reply:
Yes, I agree with using Theme values. I went to bed wondering if I wanted to do it in this PR or branch off to make reviewing less of a strain. 

Can't pass in index to determine the font as we only want to use paragon for the first of the hierarchical promos. The standard promos should all continue to use bodyCopy.

I can conditionally add it via the css prop in HierarchicalGrid.